node/overseer/src/lib.rs

```diff
@@ -457,8 +478,16 @@ where
 	/// ```
 	pub fn new(
 		leaves: impl IntoIterator<Item = BlockInfo>,
-		validation: impl Subsystem<OverseerSubsystemContext<CandidateValidationMessage>> + Send,
+		candidate_validation: impl Subsystem<OverseerSubsystemContext<CandidateValidationMessage>> + Send,
 		candidate_backing: impl Subsystem<OverseerSubsystemContext<CandidateBackingMessage>> + Send,
+		candidate_selection: impl Subsystem<OverseerSubsystemContext<CandidateSelectionMessage>> + Send,
```

Comment:
Perhaps this should be refactored out to a separate struct

Reply:
Not sure if 5e9d6699ad1113a72ef302be8afe96da33721c87 is what you meant

Reply:
Exactly what I meant